docs/modules/user-guide/partials/proc_specifying-persistent-storage.adoc

```diff
@@ -2,7 +2,7 @@
 = Specifying persistent storage
 
 [role="_abstract"]
-A `container` component can specify the volumes to be mounted on specific locations within the image. If the `container` component mounts the volume, then the volume needs a corresponding volume component of the same name. Volume names are shared across all components. Shared volume names can be used to share file systems among components.
+Use a `container` component to specify the volumes to be mounted on specific locations within the image. If the `container` component mounts the volume, then give the volume a corresponding volume component of the same name. You can share volume names across all components. Use these shared volume names to share file systems among components.
```

Comment:
The meaning is changed here:
_Volume names are shared across all components._

vs:
_You can share volume names across all components._

I suggest consulting with an SME regarding the following sentence, which doesn't make much sense:

_If the `container` component mounts the volume, then the volume needs a corresponding volume component of the same name._

Reply:
Use a container component to specify the volumes to be mounted on specific locations within the ~image~ container. 

If the container component mounts the volume, then give the volume a corresponding volume component of the same name. -> Volumes mounted in a container require a volume component with the same name.

You can share volume names across all components. -> Volume components can be shared across container components.

Use these shared volume names to share file systems among components. -> Sharing a volume across container components allow you to share files among them

Reply:
@jc-berger, please, update the text based on @maysunfaisal's input.

Reply:
@rkratky, Hey Robert, I believe I did in the commit, `final touches to revise content but not to change meaning.` See the screenshot below. Did I miss something?  

<img width="801" alt="Screen Shot 2021-06-09 at 11 22 02 AM" src="https://user-images.githubusercontent.com/70717303/121382769-ebf28300-c914-11eb-952d-a60fde5362fe.png">